specifications/cdsc-wg1-02.md

```diff
@@ -1 +1,799 @@
+* `status` - _[ClientUpdateStatus](#client-update-statuses)_ - For `type` values of `private_message` or `client_submission`, this value MUST be `complete`. For `type` values of `support_request`, this value MUST be `pending`.
+
+When Clients submit Client Updates with `type` value of `client_submission`, if the Client Update referenced in the `previous_uri` has a `status` of `open`, then the Server MUST update the `status` of that referenced Client Update to `pending`, which indicates that the Client has submitted a response for Server review.
+
+### 7.7. Modifying Client Updates <a id="clients-updates-modify" href="#clients-updates-modify" class="permalink">🔗</a>
+
+Clients may modify fields in a Client Updates object by sending an authenticated HTTPS `PATCH` request to the Client Update `uri` endpoint with the body of the request formatted a JSON object. The fields included in JSON object are the fields the Client intends to update with the submitted fields' values. If a field is not included in the `PATCH` request, the Server MUST leave the field unmodified from its current value.
+
+The following are fields that MAY be included in the `PATCH` request, and modification MUST be supported by Servers:
+
+* `read` - Servers MUST accept values of `true` and `false` from the Client.
+
+If a Client wishes to modify a previously submitted Client Update of type `client_submission` they created, the Client MUST [create a new Client Update](#clients-updates-create) with the same `previous_uri` value as the Client Update the Client is wishing to supersede. Severs MUST process newer Client Updates created by the Client as overriding the Client's previous submission.
+
+If a Client wishes to amend a previously created Client Update of type `private_message` or `support_request` they created, the Client MUST [create a new Client Update](#clients-updates-create) with the `previous_uri` value set as the Client Update `uri` for which they are wanting to amend. Severs MUST consider newer Client Updates created by the Client as amending the Client's previous support request or message.
+
+## 8. Scope Credentials API <a id="scope-creds-api" href="#scope-creds-api" class="permalink">🔗</a>
+
+To allow Clients to manage `client_secret` values used for authentication to APIs, this specification requires that Server implement a Scope Credentials API. Servers MAY implement other means of managing Scope Credentials, such as a web interface, but they MUST also mirror any Scope Credentials managed by other means on this required Scope Credentials API.
+
+The Scope Credentials API endpoints are authenticated using a Bearer `access_token` granted to Clients using OAuth's [`client_credentials` grant](https://www.rfc-editor.org/rfc/rfc6749#section-4.4) and the `client_secret` provided in the initial [Client Registration Response](#registration-response) or `client_secret` values from the [Scope Credentials API](#scope-creds-api) that include the `client_admin` scope.
```

Comment:
Do we need to have any spec for the "Client credentials grant" process that is being referenced here? 

Or are you talking about providing `access_token` AND `client_secret` or OR?